front/next.config.js

```diff
@@ -49,4 +49,14 @@ module.exports = removeImports({
       },
     ];
   },
+  webpack: (config) => {
```

Comment:
I wonder if this would work for our sequelize in types/ issue. Even if this is not what we want, I am just curious.